api/server.py

```diff
@@ -1,8 +1,24 @@
-from flask import Flask, jsonify, request, abort
+from flask import Flask, jsonify, request, abort, session
 from flask_cors import CORS
+from flask.ext.session import Session
 from util.config import REACT_PORT, API_PORT
+from util.config import SESSION_CACHE_DIR, SESSION_MODE, SESSION_THRESHOLD
+import time
 
 app = Flask(__name__)
+
+# TODO: Change if we have a database in the background
+SESSION_TYPE = 'filesystem'
+SESSION_FILE_DIR = SESSION_CACHE_DIR
+SESSION_FILE_THRESHOLD = SESSION_THRESHOLD
+# We need leading zeros on the modifier
+SESSION_FILE_MODE = int(SESSION_MODE, 8)
+SESSION_PERMANENT = True
+app.config.from_object(__name__)
+# TODO: Change for deployment, e.g. use environment variable
+app.config["SECRET_KEY"] = "grgrersg346879468"
```

Comment:
use os.urandom(24) to generate a secret key

Reply:
That's not a good idea because with this we invalidate all sessions when we restart our server.